src/Entities/Transaction.php

```diff
@@ -725,4 +725,100 @@ public function getReferenceData()
     {
         return $this->getAttribute('referenceData');
     }
+
+    /**
+     * @return bool
+     */
+    public function getIsRebill()
+    {
+        return $this->getAttribute('isRebill');
+    }
+
+    /**
+     * @return string
+     */
+    public function get3ds()
+    {
+        return $this->getAttribute('3ds');
+    }
+
+    /**
+     * @return string
+     */
+    public function getDcc()
+    {
+        return $this->getAttribute('dcc');
+    }
+
+    /**
+     * @return int
+     */
+    public function getRiskScore()
+    {
+        return $this->getAttribute('riskScore');
+    }
+
+    /**
+     * @return string
+     */
+    public function getRetryInstruction()
+    {
+        return $this->getAttribute('retryInstruction');
+    }
+
+    /**
+     * @return int
+     */
+    public function getRetryNumber()
+    {
+        return $this->getAttribute('retryNumber');
+    }
+
+    /**
+     * @return bool
+     */
+    public function getIsReconciled()
+    {
+        return $this->getAttribute('isReconciled');
+    }
+
+    /**
+     * @return bool
+     */
+    public function getHadDiscrepancy()
+    {
+        return $this->getAttribute('hadDiscrepancy');
+    }
+
+    /**
+     * @return bool
+     */
+    public function hasBumpOffer()
+    {
+        return $this->getAttribute('hasBumpOffer');
+    }
+
+    /**
+     * @return string
+     */
+    public function getBumpOffer()
+    {
+        return $this->getAttribute('bumpOffer');
```

Comment:
That doesn't correct, it should be a object

Reply:
Only return type or you mean should create bumpOffer entity before?

Reply:
Yes, it should create and return entity

Reply:
Removed this getter.